src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -81,10 +81,6 @@ class YamlFileLoader extends FileLoader
         'shared' => 'shared',
         'lazy' => 'lazy',
         'public' => 'public',
-        'abstract' => 'abstract',
-        'deprecated' => 'deprecated',
-        'factory' => 'factory',
-        'arguments' => 'arguments',
```

Comment:
Because they make no sense as "instanceof" conditionals.
(borrowed from #22294, ping @weaverryan for more info)